facade/src/main/java/org/jboss/pnc/facade/BuildTriggerer.java

```diff
@@ -36,5 +38,16 @@ int triggerBuild(int buildConfigId, OptionalInt rev, BuildOptions buildOptions)
 
     int triggerGroupBuild(int groupConfigId, Optional<GroupBuildRequest> revs, BuildOptions buildOptions)
             throws BuildConflictException, CoreException;
-    
+
+    /**
+     * Cancels a running build
+     *
+     * @param buildId ID of a running build
+     * @return True if the cancel request is successfully accepted, otherwise false.
```

Comment:
`false` means that the build doesn't exist or can it mean something else too?

Reply:
It only means, that it doesn't exist. Correcting the javadoc.